bin/stacks/routing-dashboard-stack.ts

```diff
@@ -442,6 +443,22 @@ export class RoutingDashboardStack extends cdk.NestedStack {
       }),
     })
 
+    const pairsPerChainToTrackInDashboard: [ChainId, string[]][] = [
```

Comment:
yeah that would actually make some sense, right now I'm tracking more pairs than we are displaying in the dashboard (mostly because the dashboard doesn't support all the elements tracked)